cost/aws/old_snapshots/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## v3.4
+## v3.6
+
+- Added accountName field
+
+## v3.5
```

Comment:
Missing 3.4? should this say v3.4 here and v3.5 above (instead of v3.5 and v3.6)?